### PR TITLE
feat: Adds get_pollution_forecast() docstring

### LINKED ISSUE #4: Docstring for get_pollution_forecast()

### LINKED ISSUE #5: Write about similar packages

### PR DESCRIPTION
**Pull request by christopheralex**
- [x] Add docstring
- [x] Link to man command
- [x] Updates README for similar packages

### CODE REVIEW

**Review by mel-liow — APPROVED** [review]
Looks good!

File: README.Rmd [hunk 1]
```diff
@@ -36,8 +36,7 @@ Requires a start and end date and fetches historic air pollution data for a spec
 ### `get_pollution_forecast()`
 Fetches air pollution data for the next 5 days for a specific location. The function returns a time series plot of the predicted pollution levels.
 
-
-TO ADD: a paragraph describing where your packages fit into the R ecosystem (are there any other R packages that have the same/similar functionality? Provide links to any that do. If none exist, then clearly state this as well).
+Although there is an abundance of R weather packages and APIs in the R ecosystem (e.g. owmr, ROpenWeatherMap), this particular package looks at specifically air pollution data and uses the Air Pollution API from OpenWeather. This is a unique package with functionality that (we believe) has not been made before.
```
Comment (danfke) [inline README.Rmd:39]: Thanks for this Chris!

**Review by danfke — APPROVED** [review]

File: R/get_air_pollution.R [hunk 2]
```diff
@@ -17,3 +17,22 @@
...
 
 }
+
+#' Returns a time series plot showing predicted pollutant levels for the next 5 days.
+#'
+#' Performs an API request to OpenWeather Air Pollution API,
+#' retrieves weather forecast for the next 5 days, and
+#' creates a time series graph of the pollutants with their concentration levels.
+#'
+#' @param lat A double type containing geographical latitude coordinate for the location
```
Comment (danfke) [inline R/get_air_pollution.R]: Shouldnt it be double or integer?

Comment (christopheralex) [reply R/get_air_pollution.R]: Double as discussed over slack

**Pull request merged**